models/database/stored_procs/eventsClassProcs.sql

```diff
@@ -338,6 +268,204 @@ END//
 
 
 
+-- public static function removeAddress($eventId, $address)
+    -- {
+        -- //remove an address from the event in the database
+
+        -- //categories, types, and addresses likely need their own
+        -- //methods for changes since they can have multiple entries
+    -- }
+
+
+CREATE PROCEDURE removeAddress(IN currentEventId INT, currentAddressId INT, currentAddressLine1 VARCHAR(45), currentAddressLine2 VARCHAR(45), currentCity VARCHAR(45), currentState VARCHAR(45), currentPostalCode VARCHAR(45))

```

Comment:
a lot of your sprocs are passed in extra parameters that you dont seem to be using

Reply:
Thanks for pointing that out. I'll follow up with the people who wrote this and see if we can get it corrected today.